util/GameRules.h

```diff
@@ -90,7 +90,9 @@ class FO_COMMON_API GameRules {
     //@}
 
     /** \name Mutators */ //@{
-    /** adds a rule, optionally with a custom validator */
+    /** adds a rule, optionally with a custom validator.
+        Register option setup.rules.{NAME} to override default value and
```

Comment:
Registering a rule doesn't "override" the default value, it creates the rule and sets the default value.

Reply:
Game rules and options are different things. I've removed "Register".